doc/nrf/known_issues.rst

```diff
@@ -483,60 +484,19 @@ KRKNWK-6408: ``diag`` command not supported
 Zigbee
 ======
 
-.. rst-class:: v1-9-1 v1-9-0
+.. rst-class:: v2-0-0 v1-9-1 v1-9-0
 
 KRKNWK-12937: Activation of Sleepy End Device must be done at the very first commissioning procedure for light switch sample
```

Comment:
```suggestion
KRKNWK-12937: Activation of Sleepy End Device must be done at the very first commissioning procedure for :ref:`zigbee_light_switch_sample` sample
```

Reply:
The link is in the description line, so I won't be adding it here.